Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -41,6 +41,8 @@ def warn(*args):
 if not params.get('credentials') or not (MISP_API_KEY := params.get('credentials', {}).get('password')):
     raise DemistoException('Missing API Key. Fill in a valid key in the integration configuration.')
 MISP_URL = params.get('url')
+TO_IDS = params.get('check_to_ids')
+ALLOWED_ORGS = argToList(params.get('allowed_orgs'), ',')
```

Comment:
I just want to verify, if the user did not enter any ORGS, the default value will be ",".

Is that equal to any?

Reply:
@RosenbergYehuda